README.md

```diff
@@ -4,76 +4,86 @@ This repository contains Voluntary Product Accessibility Template&reg; (VPAT&reg
 
 The VPAT is a template used to document a product's conformance with accessibility standards and guidelines. The purpose of the VPAT is to assist customers and buyers in making preliminary assessments regarding the availability of commercial "Electronic and Information Technology", also referred to as "Information and Communication Technology" (ICT) products and services with features that support accessibility.
 
-> Documents in this repository will conform to VPAT&reg; `Version 1.0`, `Version 2.2`, `Version 2.3`, `Version 2.4` or the newer `Version 2.4 Rev`. **New documents in this repository should conform to the latest standard (2.4 Rev) using the [template provided](https://github.com/springernature/vpat/blob/master/base-template.md)** 
+## Creating a new VPAT
 
-### VPAT&reg; Version 1.0
+To create a new VPAT for a product, use a *copy* of the [base template](./base-template.md) as a starting point. Ensure that you update the following placeholders within your copied template:
 
-Voluntary Product Accessibility Template&reg; (VPAT&reg;) documents for Springer Nature products, outlining Section 508 accessibility information for Springer Nature products.
+- !INSERT PRODUCT!
+- !INSERT PRODUCT NAME!  
```

Comment:
Hm, what is the "product" in contrast to the "product name"? I know, it's been like that before already, but maybe this could be improved as well?

Reply:
Hm, yes. It's a template placeholder indicating what to change in [the base template](https://github.com/springernature/vpat/blob/main/base-template.md) - so !INSERT PRODUCT! is at the very top in the title, and !PRODUCT NAME! is in the introduction section.

Not sure how I feel about this - it is a bit confusing when it's written down here stripped of context, but the product name and the product are the same thing, so it does need to be done twice... but if we only have one replacement tag for !PRODUCT! maybe people will miss one of them. 

Hm hm. Maybe what's needed is a rethink of the instructions at some point, or maybe a screenshot highlighting the bits. I'll give it some more thought, thanks!!